graphql/schema.graphql

```diff
@@ -15,6 +15,7 @@ type Query {
 
 type Mutation {
   addFavoriteProject(identifier: String!): Project!
+  removeFavoriteProject(identifier: String!): Project!
```

Comment:
Is there a need to respond with anything here?

Reply:
hmm yes i can see reasons to respond with something. The response is used to update the cache after a successful mutation. in future if discovered it's not needed then can be removed.

Reply:
My point is that we shouldn't log the event at all. I don't think there's a scenario where someone would want to know this info